### DISCUSSION TITLE
[QUESTION] moving plots

### DISCUSSION THREAD

**Discussion by nowickj** [Chia-Network/chia-blockchain#3340] [Support]
Can I move plots from one server (ubuntu 20.04) to another (new ubuntu server 20.04).
I did Cut plots to USB drive, and then Copy it to the new server. I added Plot Directory but I have a such information:

Not found Plots
Caution, deleting these plots will delete them forever. Check that the storage devices are properly connected.
Filename | Action

/home/chia/plots/plot-k32-2021-04-16-18-38-c1eb851c4dc39fa3d5514dedc7558b9b08496deeeddbdc7fcde4e3262d0f755b.plot |  
/home/chia/plots/plot-k32-2021-04-19-11-29-01f9ba57002468681b4ee1502f8f110ad5c1494fbffeb25bb16b37cd6b8b50e3.plot |  
/home/chia/plots/plot-k32-2021-04-18-18-56-a07a4ea36a82fddabd7eec41db9ce692c882fbe7956ce0138ded77d15d9d591e.plot |  
/home/chia/plots/plot-k32-2021-04-17-22-25-57ea9ad2f7864a3f471dee84849ccdebb88e49c9b8609249ec554d6521bae8c6.plot

This information is under PLOTS tab.

**Comment by garyhkirk** [comment]
Although I'm no expert (I have been researching the same thing to speed up plotting), I found a couple of videos on Youtube by searching "multiple machine chia mining" which might help (I'm not associated with any video). I think you need to verify migration when moving from one machine to another (or specifically set up plotting beforehand to be ready to migrate to the destination machine). It makes sense that one plot should belong to only one farmer.

**Comment by sargonas** [comment]
Migrating this issue to the Support section in the Discussion forums. You will be best served by moving this over there and out of the issue tracker, where the community can better assist you. (Ideally the issue tracker should be used to focus on bugs and development problems that need to be addressed, with general user support handled in Discussions and Keybase)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
